Doc/library/sqlite3.rst

```diff
@@ -1610,13 +1610,14 @@ This section shows recipes for common adapters and converters.
    sqlite3.register_converter("timestamp", convert_timestamp)
 
 
-.. _sqlite3-shortcut-methods:
+.. _sqlite3-connection-shortcuts:
```

Comment:
Just a general comment—in the future, it would be best to just leave the existing ref target in place (either instead of or addition to adding a new one), since not doing so moots much of the point of using ref targets in the first place—it requires unnecessarily changing all the existing inbound links, has a high chance of breaking some regardless (in a large doc repo like CPython that doesn't use `-n`), breaks any inbounds by external repos using intersphinx, and also breaks any normal hyperlinks linked to the `ref` target `<span>` instead of the page heading (e.g. via copying an internal inbound link).

Seems like it would be worth me opening a docs community issue or just a PR directly to the devguide to mention this (which it doesn't)? To note, the devguide currently also doesn't mention (in either the style guide or the reST Markup page under [Cross Links](https://devguide.python.org/documentation/markup/#doc-ref-role)) how ref targets should be "namespaced", i.e. by the module or page name, which is more generally useful than just repeating the basics of how to use them already covered in the Sphinx quick reference or the reST docs.

@ezio-melotti thoughts?

Reply:
Yes, sorry 'bout that :(

Reply:
I'm not sure it is worth reverting that particular change; that anchor was introduced very recently.

Reply:
Fair enough, that is an exception—though I'm not sure if it was used somewhere in a NEWS entry, etc.

Reply:
I'll have a look!

Reply:
@CAM-Gerlach:
> @ezio-melotti thoughts?

A bigger problem is that AFAIU changing ref targets also breaks anchors.  I.e., https://docs.python.org/3.12/library/sqlite3.html#sqlite3-shortcut-methods no longer works after the ref name has been changed.  For this specific case it's probably not an issue, but it's something to keep in mind when updating/moving references.

Reply:
I opened python/devguide#927 to document this